client/src/assets/sass/components/lists.scss

```diff
@@ -288,7 +288,7 @@
     margin-top: -4px;
     height: 9px;
     width: 100%;
-    background: url(https://next-geebee.ft.com/image/v1/images/raw/http%3a%2f%2fnext-geebee.ft.com%2fassets%2fbackgrounds%2ffront-page-section-header-v2.svg?source=next&format=svg&tint=000000,000000) repeat transparent;
+    background: url(https://www.ft.com/__origami/service/image/v2/images/raw/http%3a%2f%2fnext-geebee.ft.com%2fassets%2fbackgrounds%2ffront-page-section-header-v2.svg?source=next&format=svg&tint=000000,000000) repeat transparent;
```

Comment:
Any idea where this image comes from?  Does origami have a similar one we can use so we're not depending on next-geebe.ft.com?